### PR TITLE
[New] Add react-flip-move@2.9.17 w/ npm auto-update

### LINKED ISSUE #9564: [Request] Add react-flip-move
**Library name:** react-flip-move
**Git repository url:**
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #9564, cc @joshwcomeau

Pull request for issue: #9564
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/joshwcomeau/react-flip-move
 * Official website (optional, not the repository): http://joshwcomeau.github.io/react-flip-move/examples
 * NPM package url (optional): https://www.npmjs.com/package/react-flip-move
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 32
   - Count of stars: 1741
   - Count of forks: 108
 * NPM download stats (optional):
   - Downloads in the last day: 2119
   - Downloads in the last week: 11468
   - Downloads in the last month: 47980

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/react-flip-move/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-flip-move",
+  "filename": "react-flip-move.min.js",
+  "homepage": "http://joshwcomeau.github.io/react-flip-move/examples",
```
Comment (extend1994) [inline ajax/libs/react-flip-move/package.json]: Let's use `https`.

File: ajax/libs/react-flip-move/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "react-component",
+    "flip",
+    "web-animations"
+  ],
+  "npmName": "react-flip-move",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (extend1994) [inline ajax/libs/react-flip-move/package.json]: Let's use `**/!(*.gz)` to avoid `react-flip-move.min.js.gz` in old versions (v2.x).

**Comment by sashberd** [comment]
@extend1994 Why I should add versions from git? I am using NPM here. More over, I think you little but confused, those version are not on GT at all

**Comment by extend1994** [comment]
@sashberd Sorry for my mistake, those versions are only on npm. The versions needs manual adding are v0.1.5 ~ v0.1.7, v1.0.3 (on git repo).

**Comment by sashberd** [comment]
@extend1994 No need no add those versions. Was aprooved by author and @PeterDaveHello . See [here](https://github.com/joshwcomeau/react-flip-move/issues/197)

**Comment by extend1994** [comment]
@sashberd OK, next time you can mention that before I ask because I would not know you had confirm that. Thanks. I will review other parts.

**Comment by sashberd** [comment]
I know it was my mistake. Sorry

**Comment by extend1994** [comment]
No problem. 😃 It seems that you need to resolve other requests.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Pull request merged**